Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1278,39 +1285,40 @@ void ScheduleReads(u64 dvd_offset, u32 dvd_length, bool decrypt, u32 output_addr
     {
       // In practice we'll only ever seek if this is the first time
       // through this loop.
-      if (rounded_offset != head_position)
+      if (dvd_offset != head_position)
       {
         // Unbuffered seek+read
-        ticks_until_completion += CalculateSeekTime(head_position, rounded_offset);
+        ticks_until_completion += CalculateSeekTime(head_position, dvd_offset);
         DEBUG_LOG(DVDINTERFACE, "Seek+read 0x%" PRIx32 " bytes @ 0x%" PRIx64 " ticks=%" PRId64,
-                  chunk_length, rounded_offset, ticks_until_completion);
+                  chunk_length, offset, ticks_until_completion);
       }
       else
       {
         // Unbuffered read
-        ticks_until_completion += CalculateRawDiscReadTime(rounded_offset, DVD_ECC_BLOCK_SIZE);
+        ticks_until_completion += CalculateRawDiscReadTime(dvd_offset, DVD_ECC_BLOCK_SIZE);
```

Comment:
This read time appears to be using the 32kB ECC size, but in decrypt mode we're stepping `dvd_offset` by a much smaller amount each time through the loop.